uswds/uswds-site
# Update Complex Form: Progress Easily with Disabled States guidance

# Summary

Update What to do and What not to do sections with new Disabled States guidance.

> [!important]
> We need to update the changelog date before merge. 

## Related issue

Affects #2246

Does not close it as there are two other PRs that will need to be created for other pages.

Related to  #2776 

## Preview link: 

Preview link:
[Progress easily preview](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/kf-disabledstates-progresseasily/patterns/complete-a-complex-form/progress-easily/)

## Problem statement

Current page does not have guidance for disabled states.
Research and discussion among USWDS team members have resulted in a solution

## Solution

Will be adding What to do and What not to do guidance to this page as part of a Phase 1 of Disabled States work.


## Testing and review

- Make sure copy is in the correct place
- Make sure copy has no errors or typos
- Make sure links work

## !ALERT!
I believe we'll need to add changelog - this page does not appear to have one.
